### DISCUSSION TITLE
Convergence too close to initial values

### DISCUSSION THREAD

**Discussion by filippo-ciceri** [nlmixrdevelopment/nlmixr#572] [Q&A] [answered]
Hello everyone,

Trying to fit a first-order one-compartment model with the function linCmt, the estimates are very close to the initial values.  I also receive the "false convergence (8)" message. Does anyone have any suggestions to reach "better" estimates?

I am dealing with plasma concentration in the pg/mL range and amounts less than 100µg, so all values are extremely low. Could this be resposible for issues with numerical optimization?
Thank you!
Filippo

**Comment by mattfidler** [comment, marked as answer]
Hi @filippo-ciceri 

One of the nice features of `nlmixr` compared to other nonlinear mixed effect software is you can choose another optimization method to try to fit the problem.

You can try `nlmixr(..., outerOpt="bobyqa")` and see if it performs better for you.  You can specify even more types of optimization that are built in as an option too: `nlmixr(..., outerOpt="lbfgsb3c")`

It is difficult to understand what the problem with the optimization is, though.  You can also use `"saem"` instead of `"focei"`

**Comment by filippo-ciceri** [comment]
Hi @mattfidler 

The problem was actually caused by an error in the ids of the compartments. 
Thank you for the quick answer!

Filippo

**Reply by mattfidler** [reply]
The data are often the culprit in most analyses :smile:  I am glad you figured it out.
